### PR TITLE
Bare minimum to support Xcode 11

### PR DESCRIPTION
**Pull request by mbrandonw**
I just wanted to do the bare minimum to get Xcode 11 support in. I'm not deleting the Xcode project or anything. It would be great to figure out how to continue using playgrounds. Even if SwiftUI previews can help us eventually, there's still some good use cases for playgrounds (like playing with the database or router).

### CODE REVIEW

File: Sources/PointFree/Subscribe.swift [hunk 1]
```diff
@@ -8,28 +8,14 @@ import Prelude
...
-        with: get1 >>> subscribeConfirmationWithSubscribeData,
-        headersMiddleware: flash(.error, "Coupons can only be used on individual subscription plans.")
-      )
-    )
+let subscribeMiddleware: Middleware<
+  StatusLineOpen,
+  ResponseEnded,
+  Tuple2<SubscribeData?, User?>,
+  Data
+  > = requireSubscribeData
```
Comment (mbrandonw) [inline Sources/PointFree/Subscribe.swift]: this chain of middlewares was segfaulting.

someday we should try what you did for reducers on middleware. middleware is probably a lot more similar to reducers than views.

Comment (stephencelis) [reply Sources/PointFree/Subscribe.swift]: Yeah that sounds good.

File: Sources/PointFree/Subscribe.swift [hunk 2]
```diff
@@ -8,28 +8,14 @@ import Prelude
...
+    <<< validateQuanitity
```
Comment (stephencelis) [inline Sources/PointFree/Subscribe.swift]:
```suggestion
    <<< validateQuantity
```

File: Sources/PointFree/Subscribe.swift [hunk 3]
```diff
@@ -136,3 +122,39 @@ private func subscribeConfirmationWithSubscribeData(_ subscribeData: SubscribeDa
...
+  return middleware |> filterMap(
+    require1 >>> pure,
+    or: redirect(
+      with: get1 >>> subscribeConfirmationWithSubscribeData,
+      headersMiddleware: flash(.error, "Error creating subscription!")
+    )
+  )
+}
+
+private func validateQuanitity(
```
Comment (stephencelis) [inline Sources/PointFree/Subscribe.swift]:
```suggestion
private func validateQuantity(
```

File: Sources/PointFree/Subscribe.swift (hunk 2 above)
Comment (mbrandonw) [reply Sources/PointFree/Subscribe.swift]: oops didn't see this, will push to master...

**Pull request merged**